### ISSUE TITLE
Starter Templates: RTL issues

### ISSUE THREAD

**Issue by emilyaudela** [Automattic/wp-calypso#46435]
In some of the starter templates for Gutenboarding, the right-to-left elements are not correct. Please correct the right-to-left layout in these starter templates. 

Maywood

1. Starting at URL: https://wordpress.com/new/design/ar
2. Select the Maywood template
3. The heading and first paragraph or the second section is not aligned right in the preview

![capture-decran-2020-10-13-155510](https://user-images.githubusercontent.com/36699353/95935676-07f75200-0d91-11eb-883d-7354e24ace63.jpg)

Alves

1. Starting at URL: https://wordpress.com/new/design/ar
2. Select the Alves template
3. The "About" text block is not right aligned

<img width="818" alt="Screen Shot 2020-10-13 at 8 19 25 PM" src="https://user-images.githubusercontent.com/36699353/95935864-79cf9b80-0d91-11eb-974e-fd72001354e3.png">

Stratford

1. Starting at URL: https://wordpress.com/new/design/ar
2. Select the Stratford template
3. The "Services" and "Testimonial" headings should be right aligned.

<img width="744" alt="Screen Shot 2020-10-13 at 8 23 43 PM" src="https://user-images.githubusercontent.com/36699353/95936092-02e6d280-0d92-11eb-96b6-3848fd535044.png">
<img width="773" alt="Screen Shot 2020-10-13 at 8 22 37 PM" src="https://user-images.githubusercontent.com/36699353/95936094-0417ff80-0d92-11eb-99b0-b49b89d16123.png">

**Comment by andrewserong** [comment]
I think some of these are due to missing CSS rules in Varia or the child themes. @ramonjd has a PR for addressing Stratford in https://github.com/Automattic/themes/pull/2551, but I'm wondering if that same change needs to happen higher up the chain in Varia 🤔

**Comment by ramonjd** [comment]
> wondering if that same change needs to happen higher up the chain in Varia

I took a poke around and saw that there is a `rebuild-child-themes` script in the `pub/varia` directory. 🤞 we can change it in one place and recompile.

**Comment by simison** [comment]
I think this would be best handled by theme team since it affects current themes and live sites. @scruffian what do you think?

**Comment by scruffian** [comment]
Yeah lets make this change in Varia and rebuild the children. Let me know if you need any help with this.

**Comment by simison** [comment]
@scruffian Yeah, was hoping if Theme-folks could take this whole fix over?

**Comment by scruffian** [comment]
Ok, added to the backlog. Not sure when we'll get to it...

**Comment by danieldudzic** [comment]
Hi all!

Thank you for bringing this to the attention of the Theme team.

We have looked at the issue together and were able to replicate it.

We believe that the culprit here are the templates themselves.

The faulty blocks have alignment set via Block settings.

A paragraph aligned to the left should and will display as aligned to the left both in LTR and RTL.

The resolution here is not a CSS fix, but rather the removing of unnecessary alignment settings from the blocks (setting it back to `Default`).

**Comment by simison** [comment]
Nice sleuthing @danieldudzi! 👏  @ianstewart can your team review the templates here? See above comment.

**Comment by ianstewart** [comment]
@simison 👍
